korean/cafe-sungsu/cafe-sungsu.md

```diff
@@ -1 +1 @@
-아이고 집 밖에 나오니 개고생이다. 다시 [방으로 돌아간다.](../marshmallow.md) 
\ No newline at end of file
+아이고 집 밖에 나오니 개고생이다. 다시 [방으로 돌아간다.](../marshmallow.md) 

생각해 보니 국회로 가야할 것 같다. [국회로 간다.](..filibuster/filibuster.md)

```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.